services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_compose_specs.py

```diff
@@ -1,18 +1,40 @@
 import json
+import logging
 from copy import deepcopy
-from typing import Dict, List, Optional, Union
+from typing import Any, Dict, Final, List, Optional, Union
 
 import yaml
 from fastapi.applications import FastAPI
-from models_library.service_settings_labels import ComposeSpecLabel, PathMappingsLabel
+from models_library.service_settings_labels import (
+    ComposeSpecLabel,
+    PathMappingsLabel,
+    SimcoreServiceLabels,
+    SimcoreServiceSettingLabelEntry,
+)
 from settings_library.docker_registry import RegistrySettings
 
+from ...modules.director_v0 import DirectorV0Client
 from ._constants import CONTAINER_NAME
 from .docker_service_specs import MATCH_SERVICE_VERSION, MATCH_SIMCORE_REGISTRY
+from .docker_service_specs.settings import get_labels_for_involved_services
+from .scheduler.events_utils import get_director_v0_client
 
 EnvKeyEqValueList = List[str]
 EnvVarsMap = Dict[str, Optional[str]]
 
+_NANO_UNIT: Final[int] = int(1e9)
+_MB: Final[int] = 1_048_576
+# defaults to use when these values are not defined or found
+# TODO: ANE -> SAN, PC: let's make these are sensible for when a service does not define them
+DEFAULT_LIMIT_MEMORY_BYTES: Final[int] = 100 * _MB
+DEFAULT_LIMIT_NANO_CPUS: Final[int] = int(0.1 * _NANO_UNIT)
+DEFAULT_RESERVATION_MEMORY_BYTES: Final[int] = 100 * _MB
+DEFAULT_RESERVATION_NANO_CPUS: Final[int] = int(0.1 * _NANO_UNIT)
+assert DEFAULT_RESERVATION_NANO_CPUS <= DEFAULT_LIMIT_NANO_CPUS  # nosec
+assert DEFAULT_RESERVATION_MEMORY_BYTES <= DEFAULT_LIMIT_MEMORY_BYTES  # nosec
```

Comment:
I'm in the process of creating a settings for another issue which will be done through pydantic. let's sync later on.